recipes-kernel/linux/linux-fslc-imx/imx/defconfig

```diff
@@ -577,3 +577,5 @@ CONFIG_MAGIC_SYSRQ=y
 # CONFIG_DEBUG_PREEMPT is not set
 # CONFIG_DEBUG_BUGVERBOSE is not set
 # CONFIG_FTRACE is not set
+CONFIG_ION=y
+CONFIG_ION_CMA_HEAP=y
```

Comment:
I am fine in adding this but please do the same as a PR to linux-fslc repo so the original defconfig files are in sync.

Reply:
Alright, then I'll append these two lines to the linux-fslc defconfig as well.